From 2d3ae3df909931ee8fffe4561ea1422511fa9011 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Pierret=20=28fepitre=29?=
 <frederic@invisiblethingslab.com>
Date: Mon, 30 Jan 2023 11:19:15 +0100
Subject: [PATCH] Setting root is not mandatory

We enforce to have a user being an admin.
---
 pyanaconda/ui/gui/spokes/root_password.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/ui/gui/spokes/root_password.py b/pyanaconda/ui/gui/spokes/root_password.py
index 88c2f2a7a6..455821dbbf 100644
--- a/pyanaconda/ui/gui/spokes/root_password.py
+++ b/pyanaconda/ui/gui/spokes/root_password.py
@@ -168,7 +168,7 @@ class PasswordSpoke(FirstbootSpokeMixIn, NormalSpoke, GUISpokeInputCheckHandler)
     @property
     def mandatory(self):
         """Only mandatory if no admin user has been requested."""
-        return not self._users_module.CheckAdminUserExists()
+        return False
 
     def apply(self):
         if self.root_enabled and self.password:
-- 
2.45.2

